teuthology/task/internal/__init__.py

```diff
@@ -334,7 +334,7 @@ def fetch_binaries_for_coredumps(path, remote):
                     log.error(e)
                     continue
             try:
-                dump_program = re.findall("from '([^']+)'", dump_out)[0]
+                dump_program = re.findall("from '([^\\s']+)", dump_out)[0]
```

Comment:
Do you intend to match `\t`, `\n`, `\r`, `\f` or `\v` here?  It seems like the intention is to stop at a literal space character, so I'd avoid involving a character class:
```suggestion
                dump_program = re.findall("from '([^ ']+)", dump_out)[0]
```

Reply:
Yes, let's do that then.